salt/modules/state.py

```diff
@@ -285,7 +293,6 @@ def request(mods=None,
         salt '*' state.request test
         salt '*' state.request test,pkgs
     '''
-    kwargs['test'] = True
```

Comment:
I removed it at the request of @thatch45, who was reviewing the code with me as I was trying to get it to work for the integration tests.